src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/OneVersusAllTrainer.cs

```diff
@@ -364,23 +364,29 @@ private OneVersusAllModelParameters(IHostEnvironment env, ModelLoadContext ctx)
                 : base(env, RegistrationName, ctx)
         {
             // *** Binary format ***
-            // bool: useDist
+            // byte: OutputFormula as byte
```

Comment:
You are changing how model is saved / loaded from disk. For backward compatibility you will need to use versioning

Reply:
Backward compatibility isnt affected. In the old models, it stored a bool value as a byte. (0 = false = RawScores and 1 = true = Probability Normalization)
All I have done here is to add 2 = SoftMax. 

The old models will continue to be read and function as before. 


---
In reply to: [346003757](https://github.com/dotnet/machinelearning/pull/4472#discussion_r346003757) [](ancestors = 346003757)